.travis.yml

```diff
@@ -0,0 +1,39 @@
+sudo: true
+
+language: python
+
+services:
+  - docker
+
+env:
+  -
+    - TOXENV=py27
```

Comment:
I didn't mean programmer overhead. I meant overhead in Travis.

However, while I have an opinion, it isn't a strong one. If you feel strongly about this, then let's try your approach.